app/views/users/registrations/new.html.erb

```diff
@@ -20,7 +20,8 @@
       <div class="error" >
         <%= resource.errors.full_messages_for(:organisation).first %>
       </div>
-      <%= f.select :organisation_id, Organisation.all.pluck(:name,:id),{prompt:'Organisation'} %><br/>
+      <%= f.select :organisation_id, Organisation.all.pluck(:name,:id),{prompt:'Organisation'}, 
```

Comment:
@apeksha2697 , here I don't think you need to explicitly mention Organisation.all.pluck(:name, :id)
even, this will work Organisation.pluck(:name, :id)

Reply:
done